prombench/Makefile

```
@@ -0,0 +1,61 @@
+PROMBENCH_CMD        = ./prombench
```

should we include the makefile common here from the root?
```suggestion
include ../Makefile.common

PROMBENCH_CMD        = ./prombench
```